app/main.py

```diff
@@ -1,3 +1,19 @@
 def cache(func):
-    # Write your code here
-    pass
+
+    c = func.__name__
+    global cash_dict
```

Comment:
are you sure that you need that?

Reply:
Yes, since the dictionary must be stored as a global variable, which can be accessed later by the following functions